Q/QGRAF/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QGRAF"
+version = v"3.6.5"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://anonymous:anonymous@qgraf.tecnico.ulisboa.pt/v3.6/qgraf-$(version).tgz", "648cc82bf3327a4c36d36847d3921fcfa5d7f73c1646241760db030abdc86c29")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+
+cat sha256sums | while read line; do
+    name=$(echo $line | awk '{print $2}')
+    expect_sum=$(echo $line | awk '{print $1}')
+    check_sum=$(sha256sum $name | awk '{print $1}')
+    if [ $check_sum != $expect_sum ]; then
+        echo "sha256sum checks failed!"
+        echo "Expect $line but got $(sha256sum $name)!"
+        exit 1
+    fi
+done
+
+mkdir fmodules
+${FC} -o qgraf${exeext} -Os -J fmodules qgraf-3.6.5.f08
```

Comment:
```suggestion
mkdir -p "${bindir}"
${FC} -o "${bindir}/qgraf${exeext}" -Os -J fmodules qgraf-3.6.5.f08
```
Hardcoding version numbers isn't great

Reply:
Is it there any environmental variable for the version number?

Reply:
Hi, @giordano.
I cannot find any environment variables for setting the version number.
So I think that I can modify the `qgraf-3.6.5.f08` to `qgraf-*.f08`.
Do you have any comments for this modification?